spec/rester/client_spec.rb

```diff
@@ -76,12 +77,20 @@ def json_h(params)
         before { client.connect(test_url) }
 
         context 'with unsupported version' do
-          let(:test_url) { "#{RSpec.server_uri}/v2" }
+          let(:version) { 2 }
           let(:args) { ['token'] }
 
-          it 'should raise error' do
-            expect { subject.get }.to raise_error Errors::NotFoundError,
-              '/v1/tests/token'
+          it 'should be unsuccessful' do
+            expect(subject.get.successful?).to be false
+          end
+
+          it 'should have status 404' do
+            expect(subject.get.status).to eq 404
+          end
```

Comment:
Should also unit test the `Client::Response` class directly.  Maybe also put it in it's own file, even though it small?